### PR TITLE
Clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3472

### CODE REVIEW

File: SharePoint/SharePointOnline/api-access.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The API access page shows pending and approved requests. It also shows which req
     After you approve a request, it moves to the Approved requests list.
 
     > [!NOTE]
+    > Permissions of type delegated are added to the _SharePoint Online Client Extensibility Web Application Principal_ in Azure AD.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/api-access.md:49]:
```suggestion
    > Permissions of type delegated are added to the _SharePoint Online Client Extensibility Web Application Principal_ in Azure AD.
    >
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@lucabandMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by lucabandMSFT — APPROVED** [review]
approved

**Pull request merged**